app/models/user.rb

```diff
@@ -15,6 +15,8 @@ class User < ApplicationRecord
 
   before_create :generate_otp_secret
 
+  validates_format_of :mobile_number, with: /\A\d*\z/
```

Comment:
Thought it was worth some basic validation for this.

I'm not checking the length as I don't know if it will be UK numbers only!